gramps/plugins/webreport/narrativeweb.py

```diff
@@ -2424,7 +2424,7 @@ def __add_place_map_options(self, menu):
 
         mapopts = [
             [_("OpenStreetMap"), "OpenStreetMap"],
-            [_("StamenMap"), "StamenMap"],
+            [_("StamenMap"), "Stamen Map"],
```

Comment:
Not sure about this

Reply:
You could do this, but it is the first item in the list that is visible to the user, not the second.